extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -480,7 +480,8 @@ class QueueProcessor extends EventEmitter {
                     consumerReady = true;
                     const paused = options && options.paused;
                     this._consumer.subscribe(paused);
```

Comment:
The canary wont get sent. I'm thinking that a new storage location cannot already be paused because a pause request will be rejected for a non-existing storage location. So whenever the new storage location is added to the list of boostraplist, the user can then pause.

Reply:
Or actually, the canary should be sent but these consumers won't receive it, but should be added as `LAG`.
An improvement I can do is to only send canaries on initial subscribe because I don't think it will be possible to have a new storage location that is already paused